### DISCUSSION TITLE
How to use renovate only on selected Dockerfiles

### DISCUSSION THREAD

**Discussion by raphaelperrin24** [renovatebot/renovate#26541] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

latest

### If you're self-hosting Renovate, select which platform you are using.

docker

### What is your question?

In my repository I have several Dockerfiles but I only want specific once to be checked by renovate. Only if the Dockerfile contains the below line
`# renovate: registryUrl=my.private-registry.com`

I was thinking of using regexManagers to achieve this, is that possible? and how should I write the matchStrings to achieve this ?

Sorry am new to renovate.
Thanks a lot

### Logs (if relevant)

<details><summary>Logs</summary>

```

Copy/paste the relevant log(s) here, between the starting and ending backticks

```

</details>

**Comment by viceice** [comment, marked as answer]
you can achieve this on different ways. easiest it to use package rules to disable all deps and then enable only specific files via a second rule.

**Reply by raphaelperrin24** [reply]
Thanks, I achieved to do it with:

```json  
  "enabledManagers": ["dockerfile"],
  "packageRules": [
    {
      "matchDatasources": ["docker"],
      "enabled": false
    },
    {
      "matchDatasources": ["docker"],
      "matchPackagePatterns": [
        "^my.private-registry.com/.*$"
      ],
      "enabled": true,
      "versioning": "loose"
    }
  ]
```

Thanks a lot
